CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 1.1.0
```

Comment:
Aren't we looking at a major version change? The SocketConnector class had breaking changes

Reply:
Updated version and CHANGELOG for 2.0.0 in [this commit](https://github.com/atsign-foundation/socket_connector/pull/4/commits/acc28c31e9ce028dcb16948f2b3ce7c9fcec3b78)